app/controllers/videos_controller.rb

```diff
@@ -16,14 +15,17 @@ def index
   end
 
   def create
-    authorize :video
     @article = ArticleWithVideoCreationService.new(article_params, current_user).create!
 
     redirect_to @article.path + "/edit"
   end
 
   private
 
+  def autorize_video
```

Comment:
```suggestion
  def authorize_video
```

Typo

Reply:
Oh, I'm so sorry, I forgot running tests for this PR :( And thanks for review :)

Reply:
@benhalpern I think we should add `pre-commit` script for testing on every commit.